### PR TITLE
Add truncation to buildah containers|images

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by nalind** [comment]
LGTM.  We could stand to pick one or the other in terms of "do not"/"don't" and whether flags that start with "no" should be hyphenated or not, but that doesn't have to happen here.

File: cmd/buildah/containers.go [hunk 1]
```diff
@@ -17,6 +17,10 @@ var (
 			Name:  "noheading, n",
 			Usage: "do not print column headings",
 		},
+		cli.BoolFlag{
+			Name:  "no-trunc",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/containers.go]: Are you supposed to truncate no-truncate?  ;^p

Comment (rhatdan) [inline cmd/buildah/containers.go]: Fine with me.

**Comment by rhatdan** [comment]
Went with --noheading and --notruncate
and 
Do not ...

**Comment by rhatdan** [comment]
@rh-atomic-bot r+  	efad757

**Pull request closed without merging**